src/lib/actions.js

```diff
@@ -182,3 +183,42 @@ export const setUserAcceptsCookies = store.action(() => {
     // overwrite it.
   };
 });
+
+// TODO(devnook): Extract isSupportedLocale to a common format.
```

Comment:
In a follow up PR we should break up this actions.js file because it's getting quite large. I think it'd be better to have an actions folder and have individual files for the related actions.

Reply:
Filed an issue: https://github.com/GoogleChrome/web.dev/issues/3103